sktime/forecasting/base/_meta.py

```diff
@@ -80,6 +95,23 @@ def _predict_forecasters(
             for forecaster in self.forecasters_
         ]
 
+    def _update(self, y, X=None, update_params=True):
```

Comment:
Just a reminder, once [#1190](https://github.com/alan-turing-institute/sktime/pull/1190/commits) is merged, `_update` will be in `_HeterogenousEnsembleForecaster`.

Reply:
yes this is `_HeterogenousEnsembleForecaster`, I moved it there also. It will be fine with merging